server/priv/repo/migrations/20211210212804_enable_generic_subscription_claims.exs

```diff
@@ -0,0 +1,317 @@
+defmodule Realtime.RLS.Repo.Migrations.EnableGenericSubscriptionClaims do
+  use Ecto.Migration
+
+  def change do
+    execute "alter table realtime.subscription
+      drop constraint subscription_entity_user_id_filters_key cascade,
+      drop column email cascade"
+
+    execute "alter table realtime.subscription rename user_id to subscription_id"
+
+    execute "create function realtime.to_regrole(role_name text)
+      returns regrole
+      immutable
+      language sql
+      -- required to allow use in generated clause
+      as $$ select role_name::regrole $$;"
+
+    execute "alter table realtime.subscription
```

Comment:
the migration fail on this line with 

```
ERROR:  column "claims" of relation "subscription" contains null values
SQL state: 23502
```

if there are rows in `realtime.subscription`

Should there be a `truncate table realtime.subscription` at the top of the migration?

Reply:
@olirice good call! will add it in